xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -1022,7 +1022,7 @@ void CPVRClients::StartChannelScan(void)
   long perfCnt = XbmcThreads::SystemClockMillis();
 
   /* stop the supervisor thread */
-  g_PVRManager.StopUpdateThreads();
+  g_PVRManager.Stop();
```

Comment:
Ok thanks. In this case we can drop it. PVRManager does not own the addons anymore, hence it can't destroy them.